test/metabase/models/data_permissions_test.clj

```diff
@@ -116,34 +116,6 @@
           (is (nil?     (data-access-perm-value table-id-2)))
           (is (nil?     (data-access-perm-value table-id-3))))))))
 
-#_(deftest set-simple-table-permission-in-groups!-test
```

Comment:
This test was accidentally committed before, so removing it here (it's replaced by `set-new-table-permissions!-test`)